client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -340,14 +362,76 @@ class RegisteredDomainType extends React.Component {
 		);
 	}
 
-	renderAutoRenew() {
+	shouldRenderExpiringCreditCard( purchase ) {
+		return (
+			! isExpired( purchase ) &&
+			! isExpiring( purchase ) &&
+			! isOneTimePurchase( purchase ) &&
+			! isIncludedWithPlan( purchase ) &&
+			creditCardExpiresBeforeSubscription( purchase )
+		);
+	}
+
+	renderExpiringCreditCard() {
+		const { selectedSite, purchase, translate } = this.props;
+
+		if ( ! selectedSite || ! purchase ) {
+			return null;
+		}
+
+		if ( ! this.shouldRenderExpiringCreditCard( purchase ) ) {
+			return null;
+		}
+
+		const editCardDetailsPath = getEditCardDetailsPath( selectedSite.slug, purchase );
+
 		return (
-			<Card compact={ true }>
-				Auto Renew (on) <CompactFormToggle checked={ true } />
-			</Card>
+			<div>
+				<p>
+					{ translate(
+						'Your credit card is {{strong}}set to expire before your domain renewal date{{/strong}}. Please update your payment information on your account to avoid any disruptions to your service. Turn off auto-renew if you don’t want to see this message anymore.',
+						{
+							components: {
+								strong: <strong />,
+							},
+						}
+					) }
+				</p>
+				<Button primary={ true } href={ editCardDetailsPath }>
+					{ translate( 'Update your payment information' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Please update your payment information.' )` *ES Score: 8*